### PR TITLE
Fixed menu_theme and added checks for from_window

### PR DESCRIPTION
**Pull request by Loup-Garou911XD**

### CODE REVIEW

**Comment by Loup-Garou911XD** [comment]
this needs testing which i cant do rn

**Comment by Loup-Garou911XD** [comment]
i tested on 1.7.27
works perfectly fine
though the issue #212 is also applicable here so both can be fixed together

**Comment by Loup-Garou911XD** [comment]
closes #212

File: plugins/utilities/menu_theme.py [hunk 1]
```diff
@@ -1157,15 +1158,16 @@ def _make_logo(self, *args, **kwargs) -> None:
     def on_transition_in(self):
         bs.Activity.on_transition_in(self)
         random.seed(123)
-        app = bs.app
+        app = ba.app
+        env = ba.app.env
```
Comment (rikkolovescats) [inline plugins/utilities/menu_theme.py:1162]: `ba.app.env` might break on 1.7.30

**Comment by rikkolovescats** [comment]
menu theme from this pr works fine with 1.7.30?

**Comment by Freaku17** [comment]
@Loup-Garou911XD  It's broken on 1.7.26 or lower.

Works well for 1.7.30, and this warning is also seen;
```set_main_menu_window() should be passed a 'from_window' value to help ensure proper UI behavior (/storage/emulated/0/Android/data/net.froemling.bombsquad/files/mods/menu_theme.py line 1401).
```

**Comment by Loup-Garou911XD** [comment]
> menu theme from this pr works fine with 1.7.30?

Yes it works with 1.7.27 and 1.7.30 except for https://github.com/bombsquad-community/plugin-manager/issues/213
But not with 1.7.26

**Comment by Loup-Garou911XD** [comment]
> @Loup-Garou911XD  It's broken on 1.7.26 or lower. Following two errors:
> 1. >  File "/storage/emulated/0/Android/data/net.froemling.bombsquad_/files/mods/menu_theme.py", line 1168, in on_transition_in
>     vr_mode = env.vr
>               ^^^^^^
> AttributeError: 'Dummy' object has no attribute 'vr'
> 
> 2. >  File "/storage/emulated/0/Android/data/net.froemling.bombsquad_/files/mods/plugin_manager.py", line 2447, in _do_advanced
>     bui.app.ui_v1.set_main_menu_window(
> TypeError: UIV1Subsystem.set_main_menu_window() got an unexpected keyword argument 'from_window'
> 
> 
> 
> 
> Works well for 1.7.30, and this warning is also seen;
> > ```set_main_menu_window() should be passed a 'from_window' value to help ensure proper UI behavior (/storage/emulated/0/Android/data/net.froemling.bombsquad/files/mods/menu_theme.py line 1401).```

Would need something like https://github.com/bombsquad-community/plugin-manager/blob/main/plugin_manager.py#L44-L66 to fix it

**Pull request closed without merging**